src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -154,10 +195,11 @@ private bool CanMoveDown(int param)
         private bool CanUpdate(int param)
         {
             var programDataPackagePathIndex = GetIndexOfProgramDataPackagePath();
+            var appDataPackagePathIndex = GetIndexOfDefaultAppDataPackagePath();
 
             //editing builtin packages or programData package paths is not allowed.
             return RootLocations.IndexOf(Resources.PackagePathViewModel_BuiltInPackages) != param &&
-                programDataPackagePathIndex != param;
+                programDataPackagePathIndex != param && appDataPackagePathIndex != param;
```

Comment:
this one is more arguable I guess - if you repoint this path, on the next restart, you actually now have two paths - right?

I'm fine with this though.